JuliaInterface/read.g

```diff
@@ -11,6 +11,9 @@ dirs:= DirectoriesPackageLibrary( "JuliaInterface", "julia" );
 
 JuliaIncludeFile( Filename( dirs, "gaptypes.jl" ) );
 
-AddGapJuliaFuncs();
-
 ImportJuliaModuleIntoGAP( "GAP" );
+
+if not IsBound( JULIAINTERNAL_LOADED_FROM_JULIA ) then
```

Comment:
Minor (less important) idea: So, right now `JULIAINTERNAL_LOADED_FROM_JULIA` is set by some GAP code which the Julia code generates and runs... Wouldn't it make more sense to instead set such a variable from within the Julia code which we execute (or not) below? I.e. the code in `libgap.jl` could set some Julia or GAP global variable.

That would make the connection more direct.

Reply:
I do not think that is possible actually.

The variable is set within the GAP startup in `LibGAP.jl`, and there just because it makes the file loading there explicit (instead of letting the `libgap.jl` file being loaded by GAP, we load it directly.

I do not think this could be set somewhere else, actually.